spark/src/test/scala/org/apache/comet/exec/CometAggregateSuite.scala

```diff
@@ -89,6 +89,24 @@ class CometAggregateSuite extends CometTestBase with AdaptiveSparkPlanHelper {
     }
   }
 
+  // based on Spark's SQLWindowFunctionSuite test of the same name
+  test("window function: partition and order expressions") {
+    for (shuffleMode <- Seq("auto", "native", "jvm")) {
+      withSQLConf(CometConf.COMET_SHUFFLE_MODE.key -> shuffleMode) {
+        val df =
+          Seq((1, "a", 5), (2, "a", 6), (3, "b", 7), (4, "b", 8), (5, "c", 9), (6, "c", 10)).toDF(
+            "month",
+            "area",
+            "product")
+        df.createOrReplaceTempView("windowData")
+        checkSparkAnswer(sql("""
```

Comment:
What about using `checkSparkAnswerAndOperator` instead?

Reply:
With `auto` and `native` we fall back to Spark currently

Reply:
Let me see if I can conditionally use `checkSparkAnswerAndOperator` for the `jvm` shuffle case

Reply:
I updated the test. I added assertions to check for the expected shuffle exchanges and added some notes.